inc/customizer.php

```diff
@@ -142,6 +183,19 @@ function twentyseventeen_sanitize_layout( $input ) {
 }
 
 /**
+ * Sanitize the colorscheme.
+ */
+function twentyseventeen_sanitize_colorscheme( $input ) {
+	$valid = array( 'light', 'dark', 'custom' );
+
+	if ( array_key_exists( $input, $valid ) ) {
```

Comment:
This should be `in_array`, since `$valid` is not a keyed array. Right now this doesn't save `colorscheme`.